### PR TITLE
Support new PR view in `isClosedIssue`; tighten related checks

### PR DESCRIPTION
**Pull request by kovsu**

### CODE REVIEW

File: index.ts
Comment (fregante) [inline index.ts:296]:
```js
	if (!isPR()) {
```

for all

File: index.ts
Comment (fregante) [inline index.ts]: It needs to account for the other "Closed as unplanned" state

Comment (kovsu) [reply index.ts]: Hhhh, my bad. 😬

File: index.ts
Comment (kovsu) [reply index.ts]: I'm still a little sleepy. 😂

File: index.ts
Comment (fregante) [inline index.ts:310]: There are no merged issues, so this can be a single line like `return $(sel)?.textContent === 'merged'`

Same for isDraftPR. We should skip the type test where possible for this group of checks

Comment (fregante) [reply index.ts:306]: Single line

File: index.ts
Comment (kovsu) [inline index.ts:291]: Maybe use this is better.

image

Comment (kovsu) [inline index.ts:291]: Maybe use this is better.

image

Comment (fregante) [reply index.ts:291]: Let's try.

**Review by fregante — APPROVED** [review]
I'll let you merge this PR and release a new version. You can do so from the Actions tab, Releases, then type `patch` in the version field and wait for the release to appear ☺️

**Comment by kovsu** [comment]
Oh, I should clean the description input when merging.

**Comment by fregante** [comment]
`clear-pr-merge-commit-message` didn't work?

**Comment by kovsu** [comment]
It seems doesn't work. Let me check it.

image

**Comment by fregante** [comment]
It works but it's limited: https://github.com/refined-github/refined-github/issues/6611#issuecomment-2451190931

Ignore for now

File: index.ts
Comment (fregante) [inline index.ts:291]: `data-testid` is already gone :(

Test URL: https://github.com/refined-github/refined-github/pull/7987/commits

Screenshot

Comment (fregante) [reply index.ts:291]: Fixing

**Pull request merged**
